www/lang/code.js

```diff
@@ -3,7 +3,8 @@
 var Code = {};
 Code.LANGUAGE_NAME = {
 	'fr': 'Français',
-	'en': 'English'
+	'en': 'English',
+	'es': 'Espagnol'
```

Comment:
only this needs to be ¨¨Español¨¨ instead

Reply:
Solved in PR #13, sorry but I do not speak Spanish so that was based on the first Google result.

Reply:
no worries!